### PR TITLE
HTML improvements

### PR DESCRIPTION
**Pull request by Pofilo**
Fix some HTML issues in https://github.com/zeruniverse/Password-Manager/commit/8da0a5428defe0bbbbdd4e169fbba164b1629f33 and improvements in https://github.com/zeruniverse/Password-Manager/commit/c71f18919b5a401824a27ecf83400b3572ec7486 using [this tool](https://github.com/validator/validator).

### CODE REVIEW

File: src/index.php
Comment (BenjaminHae) [inline src/index.php]: `modal-footer` should be on the same level as `modal-body` so i guess the correct solution would be to make the `form`-tag embrace all the elements of `modal-content`

File: src/password.php
Comment (BenjaminHae) [inline src/password.php:23]: why was `role="navigation"` removed. This is a standard needed for example for tools helping the blind.

File: src/password.php
Comment (BenjaminHae) [inline src/password.php]: same as in index.php

File: src/index.php
Comment (BenjaminHae) [inline src/index.php]: Could you please post a screenshot of these modal-forms with the new header size? `h4` is defined by bootstrap so this could maybe lead to unwanted outcome.

File: src/password.php
Comment (Pofilo) [reply src/password.php:23]: See this: https://stackoverflow.com/a/37215942/9253338

File: src/index.php
Comment (Pofilo) [reply src/index.php]: image

File: src/index.php
Comment (Pofilo) [reply src/index.php]: You're right, I'll do that when I have some times today !

File: src/password.php
Comment (BenjaminHae) [reply src/password.php:23]: Alright!

File: src/index.php
Comment (BenjaminHae) [reply src/index.php]: It's a bit bigger, but ok for me.

**Comment by Pofilo** [comment]
About `<hx>` changes, I'm not so sure, but the rest is okay for me.

**Comment by BenjaminHae** [comment]
I'm all for it. @zeruniverse maybe you have another opinion about the `<hx>` changes?

**Comment by zeruniverse** [comment]
Sorry for the late reply. I didn't get the email notification. Don't you guys think "Use PIN to login" occupies too much space? I'm OK if `<h3>` is used but I think it should be with some style adjustment.

**Review by zeruniverse — CHANGES_REQUESTED** [review]
Please adjust the style to make the h3 take a little bit less space. Such as `<h3 style="margin:xxx">`

**Comment by Pofilo** [comment]
Okay, I'll do that during the week if I can (otherwise this week-end for sure).

**Comment by zeruniverse** [comment]
Thanks

**Comment by BenjaminHae** [comment]
I'd suggest using `<h4>` instead of `<h3>`. It may be a bit less valid as html in general, but is suggested for modals in bootstrap and I think keeping it compatible to the bootstrap defaults is a good idea.

**Comment by Pofilo** [comment]
@BenjaminHae I don't understand what you mean, I never saw any suggestion for `<h4>` instead of `<h3>`.
But I think it's still better to let `<h4>` than adding `style="margin:xxx"` so I will cancel those changes if you agree (putting `<h4>` back).

**Comment by BenjaminHae** [comment]
Thats exactly what I meant. 
Thanks

**Comment by zeruniverse** [comment]
BTW, I see multiple hx changes. Will they cause visual difference? If yes, can you upload a screenshot as well?

Thanks

**Comment by Pofilo** [comment]
Sorry for the delay, all `<hx>` changes has been reverted.

**Review by zeruniverse — APPROVED** [review]

**Comment by zeruniverse** [comment]
@Pofilo @BenjaminHae I think it's ok to merge once the conflict is solved

**Pull request merged**
